src/api/requests.js

```diff
@@ -83,8 +83,22 @@ export const publishCatchment = async (body) => {
         if (validFeatures.length === 0) {
             throw { message: "Nothing to publish"}
         }
+
+        let des 
+        if (body.details.levelId === "" && body.details.groupId.length >= 1) {
+            const groups = await fetchOrgUnitGroups()
+            des = body.details.groupId.map((g) => {
+                return groups.find((group) => group.id === g).name
+            })       
+        } else if (body.details.groupId.length === 0 && body.details.levelId !== "") {
+            const levels = await fetchOrgUnitLevels()
+            des = levels.find((level) => level.id === body.details.levelId).name
+        }
+
+        body.payload.description = Array.isArray(des) ? `groups: ${des.join(", ")} | ${body.user} | ${body.date}` : `level: ${des} | ${body.user} | ${body.date}`
```

Comment:
I think it would be good to add a comment that shows the expected format, like `groups: Clinic | brianna@crosscut.io | 2022-05-05`